builder/frameworks/component_manager.py

```
@@ -0,0 +1,226 @@
+# component_manager.py
+import os
+import shutil
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)